src/main/java/org/jabref/gui/desktop/os/NativeDesktop.java

```diff
@@ -35,7 +35,9 @@ public interface NativeDesktop {
      *
      * @return The path to the directory
      */
-     Path getDefaultFileChooserDirectory();
+     default Path getDefaultFileChooserDirectory() {
+         return Path.of(System.getProperty("user.home"), "Documents");
```

Comment:
We should check if that directory Documents is existing, otherwise I suggest we simply return user.home